### PR TITLE
Upgrade Keycloak

### PR DESCRIPTION
**Pull request by ivan-c**
- Upgrade Keycloak to latest major version (22), from 15
- Store each realm as separate JSON file (vs single file)
- Remove workarounds since fixed
  - Remove custom environment variable templating
  - Replace custom JSON logging config with new builtin flag
- Add workaround to import master realm on startup
- Use [`tini` as initialization process](https://docs.docker.com/compose/compose-file/compose-file-v2/#init)

### CODE REVIEW

**Review by pbugni — APPROVED** [review]
💯 
Nice job, fun details you had to track down.  Can't say I reviewed every line in the respective realm files, but it looks clean otherwise.

File: dev/config/keycloak/import/ltt-realm.json [hunk 1]
```diff
@@ -0,0 +1,1463 @@
+{
```
Comment (mcjustin) [inline dev/config/keycloak/import/ltt-realm.json:1]: What might I compare this with to identify what's specific to Let's Talk Tech?

**Review by mcjustin — APPROVED** [review]

File: dev/config/keycloak/import/ltt-realm.json (hunk 1 above)
Comment (ivan-c) [reply dev/config/keycloak/import/ltt-realm.json:1]: Not 100% clear if you're referring to the project name, or the cPRO config :sweat_smile: 

I copied/pasted the default `master` realm to create the `ltt` realm in this PR (65912bc), but you'll want to look at #3 for all the LTT/cPRO-specific changes

Comment (mcjustin) [reply dev/config/keycloak/import/ltt-realm.json:1]: Thanks, that realm history is what I was interested in. I wasn't referring to dhair2, but the larger project.

**Pull request merged**